### ISSUE TITLE
Vulnerable Component Ratio Missing Data

### ISSUE THREAD

**Issue by msymons** [DependencyTrack/dependency-track#368]
The "Vulnerable Component Ratio" often fails to display the 2nd integer  (denominator?).  

### Current Behavior:

The "Vulnerable Component Ratio" displays on the top-right of the dashboard.  The 2nd integer in the ratio seems to appear and disappear.  In other words, I have seen it fail to display for a while and then (I am pretty sure) it did display.  And currently it is not...

![image](https://user-images.githubusercontent.com/4938718/59353509-09d38000-8d1b-11e9-9650-df6aa02c68b4.png)

If a ratio is defined as A:B, we have A, but not B.

### Steps to Reproduce:
Navigate to dashboard and inspect.  I am not sure how to reproduce (ie, cause "B" to poof).

### Expected Behavior:
* Vulnerable Component Ratio should display correctly
* (Desired) A bit of documentation.  The word "ratio" does not appear on the docs website.

### Environment:
  
  - Dependency-Track Version: 3.5.0
  - Distribution: [ Traditional WAR]
  - BOM Format & Version: CycloneDX 1.0
  - Database Server: [ PostgreSQL ]
  - Browser:  Firefox v67.0.1 and Chrome 75.0.3770.80.

**Comment by stevespringett** [comment]
Yeah, this is a known issue. Seen it before.  I think if the quantity is large but a small percentage of the total, this issue will trigger.

I will likely remove the text from the progress bar itself so that its only a graphic and will likely create a hover that will display the percentage and ratio when hovered. Eventually, when the new front end is done, most of the charts will be dynamic, but that's a future thing.

**Comment by msymons** [comment]
Hovers/tooltip are nice, but don't they have accessibility issues?

More importantly, one cannot copy text from the DT implementation.  Example (from Dashboard and Project Overview):

![image](https://user-images.githubusercontent.com/4938718/59441258-360d0080-8df0-11e9-8d7e-7df007493159.png)

**Comment by stevespringett** [comment]
I've also thought about donut graphs as well. Regardless of what is chosen, I think we agree that the current behavior needs to improve. And yes, there are usability issues with hovers, especially on mobile.
